packages/web-components/fast-foundation/src/picker/picker.ts

```diff
@@ -558,6 +560,9 @@ export class FASTPicker extends FormAssociatedPicker {
 
         if (open && this.getRootActiveElement() === this.inputElement) {
             this.flyoutOpen = open;
+            this.closeWatcher?.destroy();
+            this.closeWatcher = new CloseWatcher();
```

Comment:
do you need to wrap this construction and the one in select in `'CloseWatcher' in window` checks?

Reply:
Ah yeah good catch